src/kiota/KiotaHost.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Kiota.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota {
+    public static class KiotaHost {
+        public static RootCommand GetRootCommand()
+        {
+            var configuration = LoadDefaultConfiguration();
+            var outputOption = new Option("--output", "The ouput path of the folder the code will be generated in.") { Argument = new Argument<string>(() => "./output") };
```

Comment:
`The ouput path of the folder the code will be generated in.` becomes
`The output directory path for the generated code files.`